UPGRADE.md

```diff
@@ -3,6 +3,44 @@ UPGRADE
 
 # From 0.14.0 to 0.15.x
 
+### Support
+
+Contact component has been renamed to Support.
+ContactBundle has been renamed to SupportBundle.
+
+Model classes and interfaces have been renamed.
+
+* Request becomes Ticket.
+* Topic becomes Category.
```

Comment:
Yeah, we were debating that. Decision was to keep things simple and name it Category. Now I start to think that maybe these few more keystrokes is worth making things clear. :D